[2025-04-05T14:33:02.417+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: descargar_y_ver_dataset.conectar_minio manual__2025-04-05T14:33:01.334605+00:00 [queued]>
[2025-04-05T14:33:02.421+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: descargar_y_ver_dataset.conectar_minio manual__2025-04-05T14:33:01.334605+00:00 [queued]>
[2025-04-05T14:33:02.421+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2025-04-05T14:33:02.421+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2025-04-05T14:33:02.422+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2025-04-05T14:33:02.426+0000] {taskinstance.py:1383} INFO - Executing <Task(PythonOperator): conectar_minio> on 2025-04-05 14:33:01.334605+00:00
[2025-04-05T14:33:02.429+0000] {standard_task_runner.py:54} INFO - Started process 153 to run task
[2025-04-05T14:33:02.431+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'descargar_y_ver_dataset', 'conectar_minio', 'manual__2025-04-05T14:33:01.334605+00:00', '--job-id', '58', '--raw', '--subdir', 'DAGS_FOLDER/ml_dust_pred_dag.py', '--cfg-path', '/tmp/tmpoea1b775']
[2025-04-05T14:33:02.431+0000] {standard_task_runner.py:83} INFO - Job 58: Subtask conectar_minio
[2025-04-05T14:33:02.432+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/***/dags/ml_dust_pred_dag.py
[2025-04-05T14:33:02.884+0000] {task_command.py:384} INFO - Running <TaskInstance: descargar_y_ver_dataset.conectar_minio manual__2025-04-05T14:33:01.334605+00:00 [running]> on host c3c93725b042
[2025-04-05T14:33:02.914+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=descargar_y_ver_dataset
AIRFLOW_CTX_TASK_ID=conectar_minio
AIRFLOW_CTX_EXECUTION_DATE=2025-04-05T14:33:01.334605+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2025-04-05T14:33:01.334605+00:00
[2025-04-05T14:33:02.915+0000] {crypto.py:84} WARNING - empty cryptography key - values will not be stored encrypted.
[2025-04-05T14:33:02.915+0000] {base.py:71} INFO - Using connection ID 'minio_s3' for task execution.
[2025-04-05T14:33:02.915+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/providers/amazon/aws/utils/connection_wrapper.py:8: DeprecationWarning: AWS Connection (conn_id='minio_s3', conn_type='s3') has connection type 's3', which has been replaced by connection type 'aws'. Please update your connection to have `conn_type='aws'`.
  #

[2025-04-05T14:33:02.916+0000] {connection_wrapper.py:303} INFO - AWS Connection (conn_id='minio_s3', conn_type='s3') credentials retrieved from login and password.
[2025-04-05T14:33:02.916+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/providers/amazon/aws/utils/connection_wrapper.py:8: DeprecationWarning: Host minio specified in the connection is not used. Please, set it on extra['endpoint_url'] instead
  #

[2025-04-05T14:33:03.039+0000] {python.py:177} INFO - Done. Returned value was: None
[2025-04-05T14:33:03.044+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=descargar_y_ver_dataset, task_id=conectar_minio, execution_date=20250405T143301, start_date=20250405T143302, end_date=20250405T143303
[2025-04-05T14:33:03.074+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2025-04-05T14:33:03.088+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
